src/App/App.tsx

```diff
@@ -138,7 +132,7 @@ export default class App extends Component<{}, AppStateInterface> {
     if (username && token) {
       storage.setItem('username', username);
       storage.setItem('token', token);
-      this.setLoggedUser(username, token);
+      this.setLoggedUser(username);
```

Comment:
Also removed here since no longer set in AppState.
I'm thinking in this case it might be good to use the `setLoggedUser()` as the single function to handle username, token, etc. therefore move the `storage.setItem()`?
Perhaps a different PR without deviating too much from this one 😄

Reply:
For now it is ok, agree with you, a new PR but I will read that logic again later on.